src/main/java/imani/game/Grid.java

```diff
@@ -27,6 +30,60 @@ public void enliven(int x, int y) {
         gameBoard[y][x] = 1;
     }
 
+    public void importRle(String rle) {
+        String[] lines = rle.split("\n");
+        int x = 0;
+        int y = 0;
+
+        for (String line : lines) {
+            if (line.startsWith("#")) {
+                continue;
+            }
+
+            if (line.startsWith("x")) {
+                Pattern pattern = Pattern.compile("x = (\\d+), y = (\\d+), rule = (.+)");
+                Matcher matcher = pattern.matcher(line);
+                if (matcher.find()) {
+                    x = 0;
+                    y = 0;
```

Comment:
Why are you setting this to 0?

Reply:
the x and y are the coordinates of the grid, my program colors the grid from the top left corner